sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -1366,8 +1367,8 @@ private void OnReceiverLinkClosed(object receiver, EventArgs e)
         public override async Task OpenLinkAsync(CancellationToken cancellationToken)
         {
             _ = await _retryPolicy.RunOperation(
-                static async (receiveLink, timeout, _) =>
-                    await receiveLink.GetOrCreateAsync(timeout).ConfigureAwait(false),
+                async (receiveLink, timeout, _) =>
```

Comment:
I think this was intentionally made static to avoid allocating. Can we pass the cancellationToken in the lambda?

Reply:
Correct this was made static to avoid the closure. The discard is the token. Just rename it to token and pass it on to the receive link?

Reply:
ah, yes.  That would be a better approach than the capture.